css_def/def.go

```diff
@@ -0,0 +1,48 @@
+package css_def
+
+import (
+	"encoding/json"
+
+	"sourcegraph.com/sourcegraph/srclib/graph"
+)
+
+func init() {
+	graph.RegisterMakeDefFormatter("Dir", newDefFormatter)
+}
+
+// DefData should be kept in sync with the def 'Data' field emitted by the
+// basic grapher.
+type DefData struct {
+	Name      string
+	Keyword   string
+	Type      string
+	Kind      string
+	Separator string
+}
+
+func newDefFormatter(s *graph.Def) graph.DefFormatter {
+	var si DefData
+	if len(s.Data) > 0 {
+		if err := json.Unmarshal(s.Data, &si); err != nil {
+			panic("unmarshal CSS def data: " + err.Error())
+		}
+	}
+	return defFormatter{s, &si}
+}
+
+type defFormatter struct {
+	def  *graph.Def
+	data *DefData
+}
+
+func (f defFormatter) Language() string { return "CSS" }
+
+func (f defFormatter) DefKeyword() string { return f.data.Keyword }
```

Comment:
Now we are using the [default]() formatter for `srclib-css` and we are using `Keyword` and `Kind`.

- `Keyword` for CSS defs, set to "selector":
![image](https://cloud.githubusercontent.com/assets/1000404/15478831/6ed1f912-20e2-11e6-815d-4eba26baa29c.png)

- `Kind` for CSS defs as well, to tell if it's a selector: "id" or "class":
![ok](https://cloud.githubusercontent.com/assets/1000404/15478809/4952faf6-20e2-11e6-8282-87ae21b34643.png)